pkg/activator/throttler.go

```diff
@@ -154,7 +154,7 @@ func (t *Throttler) UpdateCapacity(rev RevisionID, size int) error {
 	if err != nil {
 		return err
 	}
-	return t.updateCapacity(breaker, int(revision.Spec.ContainerConcurrency), size, t.activatorCount())
+	return t.updateCapacity(breaker, int(*revision.Spec.ContainerConcurrency), size, t.activatorCount())
```

Comment:
The comments here still need to be addressed. We cannot unconditionally derefence this, we'll need to resolve the value given with the value in config.

Reply:
Reading through, I'm not even sure that's true anymore 🤔

Reply:
Thanks for catching this! I've updated this to call `GetContainerConcurrency()`. `RevisionSpec` `SetDefaults(..)` will set the number to 0 if it's not set in the configs, but it's only for v1beta1